option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message CrusadeFortressReplayRequest
{
 required U64Type id = 1;
}
